article/app/pages/StoryHtmlPage.scala

```diff
@@ -27,6 +27,13 @@ object StoryHtmlPage {
     override def IE9CriticalCss: Html = stylesheetLink(s"stylesheets/ie9.$ContentCSSFile.css")
   }
 
+  def atomStyles(implicit page: Page) = Page.getContent(page) 
+    .flatMap(_.atoms)
+    .map(_.atomTypes.collect { 
```

Comment:
Do we need the double `map`?

Reply:
In this instance, I think it is justified yes.